<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountOptionsClass</class>
 <widget class="QWidget" name="AccountOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>165</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Account Name:</string>
       </property>
       <property name="buddy">
        <cstring>lneName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lneName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grbAccount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lneResource"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblExampleJabberId">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Example: romeo@example.net</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblJabberId">
          <property name="text">
           <string>Jabber ID:</string>
          </property>
          <property name="buddy">
           <cstring>lneJabberId</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lneJabberId"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblResource">
          <property name="text">
           <string>Resource:</string>
          </property>
          <property name="buddy">
           <cstring>lneResource</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblPassword">
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="buddy">
           <cstring>lnePassword</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lnePassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lblExampleResource">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Example: home; office; notebook</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lneName</tabstop>
  <tabstop>lneJabberId</tabstop>
  <tabstop>lneResource</tabstop>
  <tabstop>lnePassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
